src/paperbot/application/services/paper_dedup.py

```
@@ -3,9 +3,18 @@
 from __future__ import annotations
 
 import re
+from difflib import SequenceMatcher
 from typing import Dict, List, Optional, Tuple
 
-from rapidfuzz import fuzz
+try:
+    from rapidfuzz import fuzz
+except Exception:  # pragma: no cover - fallback when optional dependency is unavailable.
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)